episodes/audience-and-objectives.md

```diff
@@ -0,0 +1,76 @@
+---
+title: 'Target audience and teaching objectives'
+teaching: 30
+exercises: 30
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- Why is it essential to understand the target audience before starting the lesson development process?
+- How prior knowledge affects learning?
+- How do learner profiles help define the learning outcomes?
+- What are SMART objectives?
+- How is Bloom’s Taxonomy used in formulating Learning Objectives?
+- How do good objectives help with focus and provide scope for the lesson?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+By the end of this session, participants should be able to...
+
+- Explain the importance of a well-defined target audience.
+- Evaluate learner profiles and make suggestions for improvement.
+- Explain how Bloom’s taxonomy and SMART framework can help define learning objectives.
+- Evaluate lesson objectives and make suggestions for improvements.
+- Explain how the objectives are used to give the lesson more focus.
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+:::::::::::::::::::::::::::::::::::::::::::::::::: checklist
+
+## Reading List
+- [CLDT episode: Identifying Your Target Audience](https://carpentries.github.io/lesson-development-training/03-audience.html)
+- [CLDT episode: Defining Lesson Objectives/Outcomes] https://carpentries.github.io/lesson-development-training/05-objectives.html
+- Section 3 (_Formal curriculum design_) of [_Course design: Considerations for trainers - a Professional Guide_](https://doi.org/10.7490/f1000research.1118395.1) by Allegra Via et al, 2020[^1]
+- [Kirschner et al. 2006](https://carpentries.github.io/lesson-development-training/reference.html#litref)
+- [Lujan & DiCarlo 2005](https://carpentries.github.io/lesson-development-training/reference.html#litref)
+- (**Optional**) Chapter 1, How Does Students’ Prior Knowledge Affect Their Learning?, of Ambrose et al. 2010.↩︎
+- (**Optional**) Chapter 2, How Does the Way Students Organize Knowledge Affect Their Learning?, of Ambrose et al. 2010.↩︎
+
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+
+:::::::::::::::::::::::::::::::::::::::::::::::::: discussion
+
+## Discussion Questions
+- What should a good learner profile cover?
+- What are the different ways learner profiles can be used?
```

Comment:
Learner profiles are a commonly-used tool in curriculum design, and I think they are worth discussing here. But it should be noted that we do not teach currently learner profiles directly in the CLDT curriculum, nor ask trainees to write them.

Reply:
That's a good point. The exercise "EXERCISE: THINKING ABOUT TARGET AUDIENCE (15 MINUTES TOTAL FOR BOTH PARTS)" leads to a learner a profile, but we don't call it that way, and they don't write it down. Now I also see I mentioned a learner profile in one of the objectives. I will rewrite it.

Reply:
Not necessary to remove them from the discussion here altogether, but perhaps we should instead directly address ask how important it is that we do not include an activity to write profiles in the curriculum?

Reply:
Great, thank you 🙌